app/assets/javascripts/_stick-at-top-when-scrolling.js

```diff
@@ -0,0 +1,19 @@
+(function(GOVUK, GDM) {
+
+  GOVUK.GDM = GDM;
```

Comment:
Lol, I hadn't heard the monkey story before.  I guess it's another documentation/tech debt? thing.